docs/theme/mlflow/breadcrumbs.html

```diff
@@ -23,9 +23,9 @@
       <li>Search</li>
     {% endif %}
     {% if not pagename == "search" %}
-    <li class="wy-breadcrumbs-aside">
+    <!-- <li class="wy-breadcrumbs-aside">
       <a href="https://{{ github_host|default("github.com") }}/{{ github_user }}/{{ github_repo }}/blob/{{ github_version }}/{{ conf_py_path }}/{{ pagename }}{{ suffix }}" class="fa fa-github"> Edit on GitHub</a>
-    </li>
+    </li> -->
```

Comment:
I think it's fine to remove this link. We can quickly identify the RST source fix that needs to be fixed or updated by simple text search.